_data/simple-icons.json

```
@@ -3820,6 +3820,12 @@
             "hex": "ED2224",
             "source": "https://metal.equinix.com"
         },
+        {
+            "title": "Ericsson",
+            "hex": "000000",
```

The below is described as 'Ericsson Blue'.

```suggestion
            "hex": "0082F00",
```